docs/how-to/craft/enable-classic-confinement.rst

```diff
@@ -3,18 +3,174 @@
 Enable classic confinement
 ==========================
 
+When a snap needs system resources beyond what strict confinement and the available
+`interfaces <https://snapcraft.io/docs/supported-interfaces>`_ can provide, it
+needs `classic confinement <https://snapcraft.io/docs/classic-confinement>`_.
+
+This page provides guidance on how to enable classic confinement, as well as example
+solutions for snaps that have extra plugin-specific configurations to accommodate the
+confinement.
+
 
 Set the confinement level
 -------------------------
 
-.. snapcraft.yaml
+You can enable classic confinment with a single key in the project file:
 
+.. code-block:: yaml
+    :caption: snapcraft.yaml
+
+    confinement: classic
+
+When built, the snap now has admin access to the system.
```

Comment:
> admin access

Not sure how to better phrase this. The original doc said "full system access", which to me implies side effects that aren't true. I welcome suggestions

Reply:
I think "admin access" is too close to "root access" and the snap doesn't have root access.

It could be phrased like "the snap is no longer sandboxed" or "snapd runs the snap without any sandboxing or permission restrictions".

I like the sentence in the original doc that compares classic confinement to running "a traditionally packaged application", although it could be phrased like "a local executable file".

Reply:
I went with "unsandboxed access to the system": [d49dfb3](https://github.com/canonical/snapcraft/pull/5345/commits/d49dfb3e5f8e44dca56906c51ed9a8f43e9f83ac)

Reply:
I like it.